app/web/src/components/ManagementRunPrototype.vue

```diff
@@ -142,17 +142,7 @@ const lastExecution = computed<FuncRun | null>(() => {
   }
 });
 
-const lastExecutionState = computed<"ok" | "error" | null>(() => {
```

Comment:
It feels like _this part_ is the main part of the problem (fixed by `funcRunStatus`)? We were only checking 1 state field, when really there are 2 state fields to check?? (The rest of the changes seem to just be settling on the struct/type names and format?)

Reply:
Yep, that's it! One of the places used one state, the other used a different state. It was not strictly necessary to massage the shapes of history and latest into the same shape, but it *was* necessary to make sure both states were sent back in history.